node/mdSchema.ts

```diff
@@ -37,6 +37,12 @@ export const COST_CENTER_FIELDS = [
 ]
 export const COST_CENTER_SCHEMA_VERSION = 'v0.0.6'
 
+export const B2B_SETTINGS_FIELDS = [
```

Comment:
What is this used for? It feels a little odd having it in the mdSchema.ts file since it's not used for md, we might want to consider migrating it elsewhere.

Reply:
we can extract it, it's just fields needed for vbase. I can include it in the files where it's needed